sql/telemetry/buildhub2/view.sql

```diff
@@ -0,0 +1,10 @@
+-- This is an authorized view that allows us to read specific data from
```

Comment:
We should also add this to the skip list in `script/publish_views` do prevent it from being published in the schema pipeline.

Reply:
It will need to be in the dry run skiplist too, as I expect the dry run service's permissions will never allow this DDL statement to run successfully.